octokit/markdown.go

```diff
@@ -0,0 +1,46 @@
+package octokit
+
+// MardownURL is for rendering an arbitrary markdown document
+// MarkdownRawURL is for rendering raw markdown
+var (
+	MarkdownURL    = Hyperlink("/markdown")
+	MarkdownRawURL = Hyperlink("/markdown/raw")
+)
+
+// Create a MarkdownService
+func (c *Client) Markdown() (m *MarkdownService) {
+	m = &MarkdownService{client: c}
+	return
+}
+
+// A service to return HTML rendered markdown document
+type MarkdownService struct {
+	client *Client
+}
+
+// Renders a markdown document with json input
+func (m *MarkdownService) Render(uri *Hyperlink, requestParams interface{}) (renderedHTML string, result *Result) {
+	url, err := ExpandWithDefault(uri, &MarkdownURL, nil)
+	if err != nil {
+		return "", &Result{Err: err}
+	}
+	result = sendRequest(m.client, url, func(req *Request) (*Response, error) {
+		req.setBody(requestParams)
+		return req.createResponseRaw(req.Request.Post(), &renderedHTML)
+	})
+	return
+}
+
+// Renders a markdown document with string input
+func (m *MarkdownService) RenderRaw(uri *Hyperlink, markdownText *string) (renderedHTML string, result *Result) {
+	url, err := ExpandWithDefault(uri, &MarkdownURL, nil)
+	if err != nil {
+		return "", &Result{Err: err}
+	}
+	result = sendRequest(m.client, url, func(req *Request) (*Response, error) {
+		req.Header.Set("Accept", textMediaType)
```

Comment:
@feiranchen Are you sure accepting a `textMediaType` works here? If you actually test by sending a POST request to this endpoint with an ACCEPT of `text/plain;charset=utf-8`, it will respond with an error 415. Trying an ACCEPT of `application/json;charset=utf-8` or `application/vnd.github.v3.html;charset=utf-8` works fine.